src/components/Tables/TableFilterAndExtraColumnsBar.tsx

```diff
@@ -47,9 +46,9 @@ const Grow = styled.div`
     flex-grow: 1;
 `;
 
-export const filtersToComponents = <Data,>(
-    filters: Filter<Data, any>[],
-    setFilters: (filters: Filter<Data, any>[]) => void
+export const filtersToComponents = <Filter extends CoreFilter>(
```

Comment:
we need extends CoreFilter because we need to know that a filter has a filter component

Reply:
yes, although are these `any`'s okay?

Reply:
we can't use `unknown` here as Filter extends that type but would be a type with specific generic parameter i.e. that specific filter doesn't conform to `ClientSideFilter<unknown, unknown>` for example. (if i'm making sense)

so yeah happy to use any in this case.

Reply:
very interesting ty